### DISCUSSION TITLE
无法登陆专业版

### DISCUSSION THREAD

**Discussion by XINYUAN1986** [getlantern/lantern-forum-cn#853] [Q&A]
选择通过邮箱授权登录，可以收到授权码，输入授权码后，显示授权成功，点击下一步后回到主页，仍然是免费版，显示剩余流量及重置时间。

**Comment by MaxMKMax** [comment]
@XINYUAN1986 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去.

**Reply by XINYUAN1986** [reply]
你好，我在MAC系统安装软件后显示是7.9.5版本，可以登录专业版。但是在WIN7系统安装软件后显示是7.8.2版本，无法登录专业版。这种情况是WIN7系统对应的软件下载链接未更新的原因吗？

**Reply by XINYUAN1986** [reply]
@Mike777Khoo 你好，WIN7系统安装软件后显示是7.8.2版本，无法登录专业版。有解决方案吗？

**Reply by MaxMKMax** [reply]
@XINYUAN1986 您好. 我帮您问看技术部门怎么解决

**Reply by XINYUAN1986** [reply]
@Mike777Khoo **感谢答复，期待问题解决。**

**Comment by XINYUAN1986** [comment]
***@***.***
***@***.***>。*

*出现一个新的情况：：：*

*我下载了最新的MAC版本，安装成功后显示是7.9.5版本，在MAC上 - 可以 - 登录专业版。*

*我下载的WIN7版本，安装成功后显示7.8.2版本，在WIN7系统上就 - 无法 - 登录专业版。*

*WIN7系统的软件是否已更新到7.9.5版本？*

Mike K ***@***.***> 于2024年9月13日周五 01:51写道：

> @XINYUAN1986 <https://github.com/XINYUAN1986> 您好. 请您提供我支付单据和账号邮箱.
> 我帮您提交问题上去.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/853#discussioncomment-10629209>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLGIFBDV2SKO6O4XHO47RBLZWHICFAVCNFSM6AAAAABN77K34GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRSHEZDAOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> .com>
>


-- 
silence ART

**Reply by MaxMKMax** [reply]
@XINYUAN1986 
我会帮您提交问题上去.  技术部门会尽快修复.
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)


当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by XINYUAN1986** [reply]
好的，稍后我会在应用程序中提交报告，感谢答复。

**Reply by XINYUAN1986** [reply]
**报告号码：77046037**

**Reply by XINYUAN1986** [reply]
**报告号码：77092852**
